content_schemas/dist/formats/worldwide_organisation/frontend/schema.json

```diff
@@ -318,6 +326,10 @@
           "description": "Link type automatically added by Publishing API",
           "$ref": "#/definitions/frontend_links"
         },
+        "secondary_role_person": {
```

Comment:
I think the secondary and office distinction might help avoid some confusion in future so 👍